src/classes/Recipe.js

```diff
@@ -46,9 +47,13 @@ class Recipe {
   }
 
   getInstructions() {
-    const instructions = this.instructions.map(step => `${step.number}. ${step.instruction}`)
+    const instructions = this.instructions.map(step => `${step.instruction}`)
```

Comment:
We will need to remember to add back in the list style for this piece in CSS.

Reply:
This piece is connected to the HTML we actually need to do this for.

Reply:
Roger that. Issue #61 created for this.

Reply:
Looks good!